serverless/service/custom.py

```diff
@@ -0,0 +1,13 @@
+from serverless.service.types import YamlOrderedDict
+
+
+class Custom(YamlOrderedDict):
```

Comment:
what is the point of creating custom time to later convert it to dict?

Reply:
You're right. Will replace that with raw `dict`.